src/main/java/com/agilethought/atsceapi/service/HistoryServiceImpl.java

```diff
@@ -0,0 +1,43 @@
+package com.agilethought.atsceapi.service;
+
+import com.agilethought.atsceapi.dto.NewHistoryRequest;
+import com.agilethought.atsceapi.dto.NewHistoryResponse;
+import com.agilethought.atsceapi.model.History;
+import com.agilethought.atsceapi.repository.HistoryRepository;
+import lombok.extern.slf4j.Slf4j;
+import ma.glasnost.orika.MapperFacade;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+public class HistoryServiceImpl implements HistoryService{
+    @Autowired
+    private HistoryRepository historyRepository;
+
+    @Autowired
+    private MapperFacade orikaMapperFacade;
```

Comment:
`orikaMapperFacade` is never used, please move it

Reply:
Change done, I have deleted it.